### PR TITLE
Fixes on the Builder's page

### PR DESCRIPTION
**Pull request by pachicodes**
## Description
I Fixed some spelling errors and image sizing on the Builders page .

## Reviewer Notes

If there are links or steps needed to test this work, add them here.

## Related Issue(s) / Ticket(s)

If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.
* [issue]()

## Screenshot(s)

If relevant, add screenshots here.

## Use Conventional Commits

Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
standards in your pull request title and commit messages.

## Use `chore`

* for minor changes / additions / corrections to content.
* for minor changes / additions / corrections to images.
* for minor non-functional changes / additions to github actions, github templates, package or config updates, etc

```bash
git commit -m "chore: adjusting config and content"
```

## Use `fix`

* for minor functional corrections to code.

```bash
git commit -m "fix: typo and prop error in the code of conduct"
```

## Use `feat`

* for major functional changes or additions to code.

```bash
git commit -m "feat(media): creating a video landing page"
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@emotion/core": "^10.1.1",
     "@emotion/styled": "^10.0.27",
-    "@mdx-js/mdx": "^1.6.19",
+    "@mdx-js/mdx": "^1.6.22",
```
Comment (jerelmiller) [inline package.json:8]: We generally tend to update all mdx packages at once (when possible). Do you mind reverting this for now?

File: src/pages/builders.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { Button } from '@newrelic/gatsby-theme-newrelic';
 import PageLayout from '../components/PageLayout';
 import ExternalLink from '../components/ExternalLink';
 import FeatherIcon from '../components/FeatherIcon';
-import devReliquary from '../images/builders/The_Relicans.png';
+import devReliquary from '../images/builders/relicans.png';
```
Comment (jerelmiller) [inline src/pages/builders.js:8]: Was there an image at the old path? If its not used anymore, should we delete that image?

**Review by jerelmiller — CHANGES_REQUESTED** [review]
Hey @pachicodes, thanks for submitting this PR! 

I noticed a `package-lock.json` file was commited. We use `yarn` for this repo. Can you remove that file? Thanks!

File: src/pages/builders.js (hunk 2 above)
Comment (pachicodes) [reply src/pages/builders.js:8]: Yes, i believe I have deleted the image already, but I will double check.

**Comment by pachicodes** [comment]
I believe I have fixed all the requested issued. Thank you for your patience with me, I am new to Open Source and really appreciate it!☺️

**Review by jerelmiller — APPROVED** [review]
Wonderful! Thanks so much for getting that updated!!

And you're very welcome 😄. I was new to open source last year when we started the new dev site, so I know how it goes! Its definitely a journey, but a super fun space to work in. Thanks again for submitting this PR!

**Pull request merged**